coveo-settings/README.md

```diff
@@ -71,6 +71,17 @@ Matching the key of the environment variable `project.database.ssl` is done very
 - dots and underscores are ignored completely (`foo_bar` and `f__ooba.r` are equal)
     - useful for some runners that don't support dots in environment variable keys
     
+## Use ready validation
+
+You can quickly validate that a string is in a specific list like this:
+
+```python
+from coveo_settings.settings import StringSetting
+from coveo_settings.validation import InSequence
+
+ENV = StringSetting("environment", fallback="dev", validation=InSequence("prod", "staging", "dev"))

```

Comment:
I'd be willing to include this behavior through a builtin mechanism.

somewhere from Setting's init:

```
if validation is None: return _no_validation
if callable(validation): return validation
try:
    iterator = iter(validation)
except TypeError:
    raise RuntimeError('validation is not a callable and cannot be iterated")
return InSequence(*iterator)
```

This way you can forget about InSequence and still reap the benefits:

```
ENV = StringSetting("environment", fallback="dev", validation=("prod", "staging", "dev"))
```

Reply:
I'm not sure... imagine some usecases where I want something like `NotInSequence` or even `MatchRegex`

Reply:
`InSequence` remains a valid validation object because it has a __call__ method (it will pass the `is callable()` check and be used as is), so someone who wants a `MatchRegex` or `NotInSequence` would still be able to build them and specify them explicitely if they needed.

Reply:
Just to be clear; I love the approach of publishing ready made validations such as MatchRegex and NotInSequence and the like. All I'm stating is that we should handle trivial cases where the intention is clear.

Also, it's not composable unless you wrap them up with a lambda or something. Or maybe I'm confused and that's not what you meant.

`validation=['dev', 'qa']` is clear and so is `validation=re.compile(r'\d')` so I would add some magic there since it's trivial.